tools/cythonize.py

```diff
@@ -47,12 +47,6 @@
 HASH_FILE = 'cythonize.dat'
 DEFAULT_ROOT = 'scipy'
 
-# WindowsError is not defined on unix systems
-try:
-    WindowsError
-except NameError:
-    WindowsError = None
-
```

Comment:
WindowsError is also an alias of OSError, on Windows only. Also, this exception type is not used anywhere else.